src/api/mutators/useManualSchedulerMutation.test.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { ReactNode, createContext } from 'react';
+import { waitFor, act, renderHook } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { CalloutContext } from '@folio/stripes/core';
+import useManualSchedulerMutation from './useManualSchedulerMutation';
+import withIntlConfiguration from '../../test/util/withIntlConfiguration';
+
+const kyMock = jest.fn();
+
+jest.mock('@folio/stripes/core', () => ({
+  CalloutContext: createContext(null),
+  useOkapiKy: () => ({
+    post: kyMock,
+  }),
+}));
+
+describe('Automatic scheduling mutation', () => {
+  const original = console.error;
+
+  beforeAll(() => {
+    console.error = jest.fn(); // hush when we do Promise.reject
```

Comment:
what is the goal to mock `console.error`?

Reply:
It was to get rid of a console output that cluttered the test output. It looks like this, but I will remove this.
<img width="539" alt="Screenshot 2024-02-14 at 2 05 44 PM" src="https://github.com/folio-org/ui-plugin-bursar-export/assets/97154067/0a79a6c6-8a6e-4c83-88b0-a8b99a0e9c97">